bundlewatch.config.js

```diff
@@ -2,7 +2,7 @@ const bundlewatchConfig = {
   files: [
     {
       path: './dist/cloudinary-react.js',
-      maxSize: '44.5kb'
+      maxSize: '8kb'
```

Comment:
The new bundle seems to remove quite a few things from the bundle
The few things I noticed

Things previously bundled, but are now missing:
1. Inline babel code
2. React itself
3. Cloudinary-core

We need to make sure we know exactly what's being taken away from this bundle.